src/inversion_scripts/operators/TROPOMI_operator.py

```diff
@@ -231,7 +254,7 @@ def apply_tropomi_operator(
         # Get the date and hour
         iSat = sat_ind[0][k]  # lat index
         jSat = sat_ind[1][k]  # lon index
-        time = pd.to_datetime(str(TROPOMI["utctime"][iSat]))
+        time = pd.to_datetime(str(TROPOMI["time"][iSat,jSat]))
```

Comment:
This addresses the time comment below as well, but in the Operational TROPOMI data, we were defining `time` and `utctime`. The difference was that the Operational TROPOMI data (which is stored in 2 dimensions, across-track and along-track) only gives time in the along-track dimensions since it is the same for all observation across-track. So `utctime` was referenced only with the `iSat` index and now `time` is referenced with `iSat` and `jSat` and it has been expanded to be the same for the entire along-track.

I had to make this change since I don't have those dimensions for the blended product. The behavior for the operational product is not impacted.